support/cas-server-support-validation/src/main/java/org/apereo/cas/web/view/Cas30ResponseView.java

```diff
@@ -1,6 +1,7 @@
 package org.apereo.cas.web.view;
 
-import org.apache.commons.lang.StringUtils;
```

Comment:
First time I ever merged with a forked project.  I don't think I did it the ideal way either.  I'll try and see if that works merge with master works but I may need to just drop my fork.  Re fork and create a topic branch with this change.  If I need to no problem I don't want to mess up  commit history in your project.